governance/votes/Executive Vote - November 12, 2021.md

```diff
@@ -0,0 +1,84 @@
+
+$OfficeHours? This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### GUNIV3DAIUSDC-A Parameter Adjustments
+
+As per [this poll](https://vote.makerdao.com/polling/QmemHGSM?network=mainnet), the following GUNIV3DAIUSDC-A parameter adjustments will occur, if this executive proposal passes:
+
+* The [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) will be increased from 50 million DAI to 500 million DAI.
+* The [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio) will be decreased from 105% to 102%.
+* The [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee) will be decreased from 1% to 0.5%.
+
+For more information on this proposal, please see this [discussion thread](https://forum.makerdao.com/t/request-to-raise-the-guniv3daiusdc1-a-dc-to-500m/11394).
+
+### WSTETH-A Debt Ceiling Increase
+
+As per [this poll](https://vote.makerdao.com/polling/QmeQUKFm?network=mainnet), the following change will occur, if this executive proposal passes:
+
+* The [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) will be increased from 5 million DAI to 50 million DAI.
+
+For more information on this proposal, please see this [discussion thread](https://forum.makerdao.com/t/request-to-raise-staked-eth-dc-to-50m/11402).
+
+### DIRECT-AAVEV2-DAI (Aave D3M) Parameter Adjustments
+
+As per [this poll](https://vote.makerdao.com/polling/QmNbTzG1?network=mainnet#poll-detail), the following DIRECT-AAVEV2-DAI parameter adjustments will occur, if this executive proposal passes:
+
+* The [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) will be increased from 10 million DAI to 50 million DAI.
+* The [Target Available Debt (gap)](https://makerdao.world/en/learn/governance/module-dciam) will be increased from 10 million DAI to 25 million DAI.
+* The Target Borrow Rate will be decreased from 4% to 3.9%.
+
+For more information on this proposal, please see this [discussion thread](https://forum.makerdao.com/t/d3m-dc-increase-target-borrow-rate-decrease-proposal/11421). Further information regarding these parameters can also be found in this [discussion thread](https://forum.makerdao.com/t/discussion-direct-deposit-dai-module-d3m/7357).
```

Comment:
Wondering if the second sentence here should start "Further information on DIRECT-AAVEV2-DAI (Aave D3M) can also be found..."